### PR TITLE
Fixed $tabularData on overview.php

### PR DESCRIPTION
**Pull request by DRSDavidSoft**
Apparently, In order for the latest versions of Kint to show 2-d arrays in tabular-view, no columns should have more than 50% empty cells .

### CODE REVIEW

**Comment by DRSDavidSoft** [comment]
Fixes #219.

**Comment by DRSDavidSoft** [comment]
Is this PR still relevant for Kint 2.0 examples? Or the new plugin replaces this?

**Comment by jnvsor** [comment]
The new plugin replaces it. It's also more strict than the old behaviour so it'll likely make your issue in #219 worse, though you can mitigate that by writing your own plugin for it.

If you don't find any massive explosive issues with it, mind if I close the PR and issue?

**Comment by DRSDavidSoft** [comment]
I don't mind at all, but IMO it's more appropriate if you push it to the 1.0 branch for historical versions (some future visitors might still be wanting to use the older version of kint.)

It's just an example script, after all.

**Comment by jnvsor** [comment]
Ah true, I thought this was tweaking behaviour not an example. Though I do spy a bug right off: The day in the dates will go up to `2013-01-011` now. Lower the condition to 10 or so and it should be fine

**Comment by DRSDavidSoft** [comment]
You have eagle eyes! I hope you accept this solution, I tried to keep it still readable.

File: examples/overview.php [hunk 1]
```diff
@@ -115,16 +115,17 @@ public function ensureUser() { $this->user->ensure(); }
 $user->setCreatedDate( new DateTime( '2013-10-10' ) );
 $userManager = new UserManager();
 
-for ( $i = 1; $i < 6; $i++ ) {
+for ( $i = 1; $i < 12; $i++ ) {
+	$number = str_pad($input, 2, "0", STR_PAD_LEFT);
```
Comment (jnvsor) [inline examples/overview.php]: I spy with my little eye, copy pastey pastey :)

**Review by jnvsor — CHANGES_REQUESTED** [review]

File: examples/overview.php (hunk 1 above)
Comment (jnvsor) [reply examples/overview.php]: In case you still don't see it: `$i != $input`

Comment (DRSDavidSoft) [reply examples/overview.php]: That's actually a misplaced auto completion. That's what happens if I hurry to commit :D

**Review by DRSDavidSoft — COMMENTED** [review]
Done. Sorry about multiple commits!

**Comment by DRSDavidSoft** [comment]
Everything good for the PR now?

**Comment by jnvsor** [comment]
No problem - I'll squash them in the end. Just let me test this quickly

**Review by jnvsor — APPROVED** [review]

**Comment by jnvsor** [comment]
Squashed as 425c6127b54e47934f36d69519342c94d2d1c911

**Pull request closed without merging**